spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/SCConfigService.java

```diff
@@ -202,18 +204,26 @@ private void registerDevice() {
             public void call(Integer integer) {
                 if (integer == 1) {
                     try {
-                        // first check if device is already registered
-                        String response = backendService.get(String.format(deviceEndpoint, getClientId()));
-                        if (response.equals("null")) {
-                            backendService.post(registrationEndpoint,
-                                    String.format("{\"identifier\": \"%s\", \"device_info\": {\"os\":\"%s\"} }",
-                                            getClientId(), getAndroidVersion()
-                                    )
-                            );
-                        }
-                        else {
-                            Log.d(LOG_TAG, "Device is already registered.");
-                        }
+                        HttpHandler.getInstance().get(String.format(deviceEndpoint, getClientId()))
```

Comment:
Let's get rid of this b/c the backend service is already doing the registration over mqtt